qdrant-landing/content/blog/qdrant-1.10.x.md

```diff
@@ -0,0 +1,204 @@
+preview_image: /blog/qdrant-1.10.x/social_preview.png
+social_preview_image: /blog/qdrant-1.10.x/social_preview.png
+date: 2024-06-25T00:00:00-08:00
+author: David Myriel
+featured: false 
+tags:
+  - vector search
+  - ColBERT late interaction
+  - BM25 algorithm
+  - search API
+  - new features
+---
+
+[Qdrant 1.10.0 is out!](https://github.com/qdrant/qdrant/releases/tag/v1.10.0) This version introduces at some major changes, so let's dive right in:
+
+**Universal Query API:** All search APIs are now consolidated into the single Query API endpoint.</br>
+**Hybrid Search:** Hybrid Search is offered out of the box, and you can use it via Query API.</br>
+**BM42 Algorithm:** Alternative method of scoring is best suited for Hybrid Search on short docs.</br>
+**Multivector Support:** Native support for late interaction ColBERT is accessible via Query API.
+
+## One Endpoint for All Queries
+**Query API** will consolidate all search APIs into a single request. You can now configure a single request to `nearest`/`fusion` search or `discover`/`recommend` or `context`/`order_by` results. Previously, you had to work outside of the API to combine different search requests. Now these approaches are reduced to parameters of a single request, so you can avoid merging individual results.
+
+For example, you can configure Query API to run Discovery search. Let's see how that looks:
+
+```http
+POST collections/{collection_name}/points/query
+{
+  "query": {
+    "discover": {
+      "target": <vector_input>,
+      "context": [{"positive": <vector_input>, "negative": <vector_input>}]
+    }
+  }
+}
+```
+
+We will be publishing examples in docs and our new API specification. *For more details, read the [Query API documentation] (/documentation/concepts/query/)*
+
+## Native Hybrid Search Support
+Query API will also natively support **Hybrid Search**. Up to this point, you had to combine the results via `fusion` of `sparse` and `dense` vectors on your own. This is now sorted on the back-end, and you only have to configure them as basic parameters for Query API. 
+
+```http
+POST /collections/{collection_name}/points/query
+{
+    "prefetch": [
+        {
+            "query": { 
+                "indices": [1, 42],    // <┐
+                "values": [0.22, 0.8]  // <┴─sparse vector
+             },
+            "using": "sparse",
+            "limit": 20,
+        },
+        {
+            "query": [0.01, 0.45, 0.67, ...], // <-- dense vector
+            "using": "dense",
+            "limit": 20,
+        }
+    ],
+    "query": { "fusion": "rrf" }, // <--- reciprocal rank fusion
+    "limit": 10
+}
+```
+Keep in mind that the Query API can now perform **sub-requests**. You will need to define a strategy to merge these requests using new parameters. For example, you can now include reranking within Hybrid Search, which can open door to strategies like iterative refinement via matryoshka embeddings. 
```

Comment:
"Keep in mind" sounds like a warning note, while this is a feature. Or were you trying to warn against not specifying a `query`?